src/server/telemetry.js

```diff
@@ -0,0 +1,168 @@
+var http = require('http');
+var https = require('https');
+var _ = require('../utility');
+var urlHelpers = require('./telemetry/urlHelpers');
+
+var defaults = {
+  network: true,
+  networkResponseHeaders: false,
+  networkRequestHeaders: false,
+  log: true
+};
+
+function Instrumenter(options, telemeter, rollbar) {
+  this.options = options;
+  var autoInstrument = options.autoInstrument;
+  if (options.enabled === false || autoInstrument === false) {
```

Comment:
Nit: do we want to support truthy values, rather than strict boolean? I can see someone potentially doing something like this: `options.enabled = someThing.length`.

Reply:
This does support truthy, and significantly, it also treats `null` or `undefined` as truthy. In other words, omitting the option lets it be enabled by default.

Both the interface spec and the implementation are based on the existing browser version of this feature. Given there's more than one way to do this, I preferred consistency over other possible ideas.